example/lib/wifi_screen/wifi_bloc.dart

```diff
@@ -58,9 +58,6 @@ class WiFiBlocSoftAP extends Bloc<WifiEvent, WifiState> {
   Stream<WifiState> _mapProvisioningToState(
       WifiEventStartProvisioningSoftAP event) async* {
     yield WifiStateProvisioning();
-    List<int> customData = utf8.encode("Some CUSTOM data\0");
```

Comment:
Hi Felix, 

Thanks for sending PR, in case the developer needs to send custom data to the device, they can use these lines. Are you sure this is the root cause of the bug for the esp32s2?  Even if so, we can't remove this functionality. It should be debugged in another way.

Reply:
Could it be that my ESP aborts the provisioning when it receives data it doesn't expect? I haven't set up any custom endpoint for receiving custom data